recipe/session/session.go

```diff
@@ -48,14 +48,9 @@ func makeSessionContainerInput(accessToken string, sessionHandle string, userID
 func newSessionContainer(config sessmodels.TypeNormalisedInput, session *SessionContainerInput) sessmodels.SessionContainer {
 
 	revokeSessionWithContext := func(userContext supertokens.UserContext) error {
-		success, err := (*session.recipeImpl.RevokeSession)(session.sessionHandle, userContext)
-		if err != nil {
-			return err
-		}
-		if success {
-			clearSessionFromCookie(config, session.res)
-		}
-		return nil
+		_, err := (*session.recipeImpl.RevokeSession)(session.sessionHandle, userContext)
+		clearSessionFromCookie(config, session.res)
+		return err
```

Comment:
if there is an `err`, you don't want to clear the cookies.. as the err might have happened cause the core is down?

Reply:
yes, that's correct. Updated the fix.